playbooks/group_vars/zoweNode.yml

```diff
@@ -2,8 +2,11 @@
 
 ansible_port: 22
 
+
 ################################################################################
 # variables for zowe uninstallation and installation
+# server tn3270 port
+zowe_zlux_terminal_telnet_port: 623
```

Comment:
I believe this is wrong port for your system. For your system, you can remove this line to fall back to default port 22.